src/Scripting/Core/XmlFileResolverForAssemblies.cs

```diff
@@ -62,7 +60,7 @@ public static bool TryFindXmlDocumentationFile(string assemblyFilePath, out stri
             // 3. Look for reference assemblies
 
             string referenceAssemblyFilePath;
-            if (!TryGetReferenceFilePath(assemblyFilePath, out referenceAssemblyFilePath))
+            if (!Roslyn.Utilities.ReferencePathUtilities.TryGetReferenceFilePath(assemblyFilePath, out referenceAssemblyFilePath))
```

Comment:
if-def'd code. `Roslyn.Utilities` namespace is needed only where this code is active and not anywhere outside of the if-def'd region.. Also, the only occurrence, so just qualified in place.